### ISSUE TITLE
Supported/Tested Hardware List

### ISSUE THREAD

**Issue by RedlineTriad** [cujomalainey/ant-rs#10]
There are various ant usb sticks, and it would be interesting to have a list of tested/supported/recommended hardware options + OS combinations?

**Comment by cujomalainey** [comment]
Good point, right now the only sticks I have tested are ANT-m sticks. In theory all sticks should behave the same at the protocol level (I am not certain about USB).

Regarding OS, most systems should be covered as I believe libusb is support by almost all major OS.

It honestly might be easier to have a "doesn't work" list if there are known bad combinations, and even then I think it would be an easy fix.

**Comment by harrychin** [comment]
The only difference between the USB1, USB2, USB-m, and USB-UIF boards are the features they support and the serial baud rates. The serial rates might need to be tweaked for USB1 but I doubt there are many out there in the wild.

**Comment by cujomalainey** [comment]
Leaving this open until we get around to deleting the buggy Linux uart driver.
